addons/website_livechat/static/src/new/core/persona_service.js

```diff
@@ -24,6 +24,8 @@ patch(PersonaService.prototype, "website_livechat", {
                     type: "partner",
                 });
             }
+            persona.displayName =
+                persona.partner?.name ?? persona.partner?.displayName ?? persona.name;
```

Comment:
I feel like this should be computed inside the Persona model ? Guest are missing

Reply:
Yeah sure, it was but actually we took the opposite direction in https://github.com/odoo-dev/odoo/pull/2110 (which leads to shadowing the existing getter) :man_shrugging: